whois-api/src/main/java/net/ripe/db/whois/api/mail/dequeue/BouncedMessageParser.java

```diff
@@ -185,7 +185,8 @@ private List<String> extractRecipients(final DeliveryStatus deliveryStatus) {
             }
             final Matcher finalRecipientMatcher = FINAL_RECIPIENT_MATCHER.matcher(recipient);
             if (!finalRecipientMatcher.matches() || finalRecipientMatcher.groupCount() != 2){
-                LOGGER.error("Wrong formatted recipient {}", recipient);
+                LOGGER.info("Wrong formatted recipient {}", recipient);
+                LOGGER.error("Wrong formatted recipient <Filtered>");
```

Comment:
No need to mention `<Filtered>` just include the text `Wrong formatted recipient`. Can you include the message-id to make debugging easier ?

Reply:
I don't have message id there but to make debugging easier I also added the message as info

Reply:
Thanks

Reply:
Actually I see sentry does show debug messages, should we change this behaviour ?